doc/ref_arch/kubernetes/README.md

```diff
@@ -36,17 +34,6 @@ This is Kubernetes based Reference Architecture (RA-2)
 | Chapter 07 | Lots of SME feedback |
 | Appendix B - Guidance For workload isolation (Multitenancy) with Kubernetes for application vendors | Still developing content |
 
-## Table of Contents
```

Comment:
why delete these as these are easy traversal links for folks working in Github?

Reply:
Fixed in 3abd355